Pushbullet/Web/pushbulletnotifications.html

```diff
@@ -0,0 +1,48 @@
+<div id="PushbulletConfigurationPage" data-role="page" class="page type-interior pluginConfigurationPage" data-controller="__plugin/pushbulletnotificationsjs">
+    <div data-role="content">
+        <div class="content-primary">
+            <form class="PushbulletConfigurationForm">
+                <div style="height:0; overflow: hidden;">
+                    <input type="text" name="fakeusernameremembered" tabindex="-1" />
+                    <input type="password" name="fakepasswordremembered" tabindex="-1" />
+                </div>
+                <div class="selectContainer">
+                    <select is="emby-select" id="selectUser" label="Configure Pushbullet notifications for:"></select>
+                </div>
+                <label class="checkboxContainer">
+                    <input is="emby-checkbox" type="checkbox" id="chkEnablePushbullet" />
+                    <span>Enabled</span>
+                </label>
+                <div class="inputContainer">
+                    <input is="emby-input" type="text" id="txtPushbulletChannel" label="Pushbullet channel:" />
+                    <div class="fieldDescription">
+                        Pushbullet channel
+                    </div>
+                </div>
+                <div class="inputContainer">
+                    <input is="emby-input" type="text" id="txtPushbulletAuthKey" required="required" label="Pushbullet auth token:" />
+                    <div class="fieldDescription">
+                        Auth key for your application
+                    </div>
+                </div>
+                <div class="inputContainer">
+                    <input is="emby-input" type="text" id="txtPushbulletDeviceId" label="PushBullet Device Id:" />
+                    <div class="fieldDescription">
+                        Device Identification
+                    </div>
+                </div>
```

Comment:
Had a quick look as to why this was missing in the 'old' config page. Seems like this is a currently completely ignored setting (except for some debug logging). However, I am not sure if we should remove it or add  it to the actual requerst sent to Pushbullet (out of scope for this PR ofc).

Ref:
* as commented in the options class: https://github.com/jellyfin/jellyfin-plugin-pushbullet/blob/fe2f27a98c21c81bdcf85c6f1f43fc3c9773ea22/Pushbullet/Configuration/PushbulletOptions.cs#L20
* and its only use so far: https://github.com/jellyfin/jellyfin-plugin-pushbullet/blob/fe2f27a98c21c81bdcf85c6f1f43fc3c9773ea22/Pushbullet/Notifier.cs#L76

Reply:
@h1dden-da3m0n i notice that may be hide it for now or removing completely. b/c i added that the script need to get id **txtPushbulletDeviceId** value.

Reply:
I will bring it up in our matrix to see if anyone knows why it is even still there and we decide based on what I can dig up

Reply:
@grafixeyehero Okay, based on internal discussions we think its better to remove this before users get confused why the option does nothing

Reply:
@h1dden-da3m0n done https://github.com/jellyfin/jellyfin-plugin-pushbullet/pull/28/commits/6d77f7f4764a1a1225c27abbbf2616c9d7df3f72